.github/workflows/auto-merge-dependabot.yml

```
@@ -0,0 +1,28 @@
+on:
+    pull_request:
+        types: [opened, synchronize]
+
+jobs:
+    auto-merge:
+        if: github.actor == 'dependabot[bot]'
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check if PR is safe to auto-merge
+              id: check
+              run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)